python/cudf/cudf/_lib/cpp/io/types.pxd

```diff
@@ -32,6 +32,10 @@ cdef extern from "cudf/io/types.hpp" \
         BROTLI "cudf::io::compression_type::BROTLI"
         ZIP "cudf::io::compression_type::ZIP"
         XZ "cudf::io::compression_type::XZ"
+        ZLIB "cudf::io::compression_type::ZLIB"
+        LZ4 "cudf::io::compression_type::LZ4"
+        LZO "cudf::io::compression_type::LZO"
+        ZSTD "cudf::io::compression_type::ZSTD"
```

Comment:
There's significant divergence between the set of compressions supported at the C++ level (as defined by this enum) and the ones exported to Python (in orc.pyx). Is that based on the what compression types are supported by a given file format (e.g. ORC) or are we just choosing not to expose all features?

Reply:
Yeah, this covers all compression types in all of cuIO. Different formats only support a subset (and we support a subset of that).